stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -116,6 +116,7 @@ class AssignManualQuantsLines(models.TransientModel):
     _name = "assign.manual.quants.lines"
     _description = "Assign Manual Quants Lines"
     _rec_name = "quant_id"
+    _order = "lot_id, location_id, package_id"
```

Comment:
I think you should sort first by location

Reply:
Done :)